extension/content.js

```diff
@@ -293,6 +293,39 @@ function indentInput(el, size = 4) {
 	el.selectionEnd = el.selectionStart = selectionStart + indentationText.length;
 }
 
+function addFileCopyButton() {
```

Comment:
No problem. Do we want to start doing this in general for new features that are more than a few lines?